dashboard/src/features/transactions/actions.js

```diff
@@ -70,16 +70,39 @@ function getTemplateXpubs(tpl) {
 }
 
 form.submitForm = (formParams) => function(dispatch) {
-  const buildPromise = chainClient().transactions.build(builder => {
-    const processed = preprocessTransaction(formParams)
+  try {
+    const buildPromise = chainClient().transactions.build(builder => {
+      const processed = preprocessTransaction(formParams)
```

Comment:
If `preprocessTransaction` is the issue, we don't need to wrap our existing Promise-returning function in a giant try catch block. Can this just wrap line 74 instead?